.vscode/snippets.code-snippets

```diff
@@ -0,0 +1,21 @@
+{
+  "Tailwind Component": {
+    "scope": "javascript,typescript",
+    "prefix": ["component", "@customElement"],
+    "body": [
+      "import { TailwindElement } from \"@/classes/TailwindElement\";",
+      "import { localized } from \"@lit/localize\";",
+      "import { html } from \"lit\";",
```

Comment:
Nitpick: can we order external library imports first?

Reply:
Yeah! We could even set up ESLint rules for that: https://github.com/import-js/eslint-plugin-import

Reply:
Done! The template now properly orders the imports